extras/coverage-report.sh

```diff
@@ -1,7 +1,18 @@
-# Run tests, generate coverage report and open it on a browser
+#!/bin/sh
+# Run tests, generate coverage report and open it in a browser
 #
 # Requires: coverage 3.3 or above from http://pypi.python.org/pypi/coverage
 
-coverage run --branch $(which trial) --reporter=text tests
-coverage html -i
+trial="`which trial`"
```

Comment:
When I last checked it, twisted trial didn't work in Python 3 (unlike pytest-twisted). See http://twistedmatrix.com/trac/ticket/5965. We switched to pytest-twisted because of this. That's why I'm not comfortable merging it :)